src/test/java/com/redhat/red/build/koji/model/xmlrpc/messages/RpmBuildListResponseTest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc.messages;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import com.redhat.red.build.koji.model.xmlrpc.KojiBuildInfo;
+import com.redhat.red.build.koji.model.xmlrpc.KojiRpmInfo;
+
+import static junit.framework.TestCase.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class RpmBuildListResponseTest
+                extends AbstractKojiMessageTest
+{
+    @Test
+    public void verifyVsCapturedHttp() throws Exception
+    {
+        RpmBuildListResponse response = parseCapturedMessage( RpmBuildListResponse.class, "getLatestRPMS-response.xml" );
+
+        List<KojiRpmInfo> rpms = response.getRpmBuildList().getRpms();
+        List<KojiBuildInfo> builds = response.getRpmBuildList().getBuilds();
+
+        assertEquals( 2, rpms.size() );
+        assertEquals( 1, builds.size() );
+
+        KojiRpmInfo rpm1 = rpms.get( 0 );
+        KojiRpmInfo rpm2 = rpms.get( 1 );
+        KojiBuildInfo build1 = builds.get( 0 );
+
+        assertEquals( Integer.valueOf( 6777603 ), rpm1.getId() );
+        assertEquals( Integer.valueOf( 6777602 ), rpm2.getId() );
+        assertEquals( 848604, build1.getId() );
+    }
+
+    @Test
+    @Ignore
+    public void roundTrip() throws Exception
+    {
+        KojiRpmInfo a1 = new KojiRpmInfo( 6777603, 848604, "rhmap-fh-openshift-templates", "4.7.4", "2.el7", "noarch" );
+        KojiRpmInfo b1 = new KojiRpmInfo( 6777602, 848604, "rhmap-fh-openshift-templates", "4.7.4", "2.el7", "noarch" );
+        List<Object> one = Arrays.asList( a1, b1 );
+
+        KojiBuildInfo a2 = new KojiBuildInfo( 848604, 59108, "rhmap-fh-openshift-templates", "4.7.4", "2.el7" );
+        List<Object> two = Arrays.asList( a2 );
+
+        List<List<Object>> objs = Arrays.asList( one, two );
+
+        // FIXME: Fix this test
```

Comment:
The parsing here needs to be fixed. The response is a list containing two lists. The first list is a list of RPMs (`KojiRpmInfo`) and the second list is the list of builds containing those RPMs (`KojiBuildInfo`). See ` RpmBuildListResponse(List<List<Object>>)`.

Reply:
I guess this is fixed? because I see all the tests passed.

Reply:
No, there is an `@Ignore` there that has to be removed.

Reply:
I removed the `@Ignore` in bb7c937 and the test now fails.

I believe that `ExternalKojiRpmTest` shows the client method to be working properly.

I believe that the test failure has something to do with how the data is received from the server as JSON maps vs. when the data is already parsed from the original JSON maps into Koji objects before being passed to the constructor in the test itself.